src/lib/mina_compile_config/dune

```diff
@@ -1,7 +1,7 @@
 (library
  (name mina_compile_config)
  (public_name mina_compile_config)
- (libraries currency)
+ (libraries)
  (preprocessor_deps ../../config.mlh)
```

Comment:
I might be misreading this, but if the libraries stanza is empty without the currency module, then do we even need it. Can it just be omitted?

Reply:
I believe the stanza could be omitted, but I kind of like having it, to demonstrate that there are no dependencies.